src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.ts

```diff
@@ -100,7 +100,7 @@ export class PoolFormComponent implements OnInit {
     this.form = new CdFormGroup(
       {
         name: new FormControl('', {
-          validators: [Validators.pattern('[\\.A-Za-z0-9_-]+'), Validators.required]
+          validators: [Validators.pattern(/[\.A-Za-z0-9_\-/]+/), Validators.required]
```

Comment:
The ``-`` does not have to escaped (https://javascript.info/regexp-escaping). I think to remember that it is common practice to put the ``-`` to the end char list.